content/en/docs/reference/kubectl/overview.md

```diff
@@ -101,48 +101,58 @@ Remember: For more about command operations, see the [kubectl](/docs/user-guide/
 
 ## Resource types
 
-The following table includes a list of all the supported resource types and their abbreviated aliases:
-
-Resource type    | Abbreviated alias
--------------------- | --------------------
-`apiservices` |
-`certificatesigningrequests` |`csr`
-`clusters` |
-`clusterrolebindings` |
-`clusterroles` |
-`componentstatuses` |`cs`
-`configmaps` |`cm`
-`controllerrevisions` |
-`cronjobs` |
-`customresourcedefinition` |`crd`
-`daemonsets` |`ds`
-`deployments` |`deploy`
-`endpoints` |`ep`
-`events` |`ev`
-`horizontalpodautoscalers` |`hpa`
-`ingresses` |`ing`
-`jobs` |
-`limitranges` |`limits`
-`namespaces` |`ns`
-`networkpolicies` |`netpol`
-`nodes` |`no`
-`persistentvolumeclaims` |`pvc`
-`persistentvolumes` |`pv`
-`poddisruptionbudget` |`pdb`
-`podpreset` |
-`pods` |`po`
-`podsecuritypolicies` |`psp`
-`podtemplates` |
-`replicasets` |`rs`
-`replicationcontrollers` |`rc`
-`resourcequotas` |`quota`
-`rolebindings` |
-`roles` |
-`secrets` |
-`serviceaccounts` |`sa`
-`services` |`svc`
-`statefulsets` |
-`storageclasses` |
+The following table includes a list of all the supported resource types and their abbreviated aliases.
+
+(This output can be retrieved from `kubectl api-resources`, and is accurate as of Kubernetes 1.13.3.)
+
```

Comment:
@justaugustus, one nit:
- Not sure about setting the accuracy of the output with a specific K8s version. I generally think that the implied version of tools/libraries is equal to the published version of the docs, unless stated. After 1.1.4 is published, will this table change?
- Possibly remove the parens and change to something like:
You can view the list of supported resource types by calling `kubectl api-resources`.

Reply:
@kbhawkey -- I tied the accuracy to the release version, because I know this will have the potential to fall out of date based on APIs that either get introduced or deprecated. I'm not sure how updates like this are normally handled or who is truly responsible for maintaining them.

Should I change the disclaimer to something like, "This table is accurate as of writing, but the list of resources can/will change as APIs are introduced or deprecated. Up-to-date information about available resources can be retrieved using the `kubectl api-resources` command."?

Reply:
@justaugustus, OK, I agree with @zacharysarah about using a shortcode for the version.
Something like this may work: 
This table is accurate as of Kubernetes version {{< latest-version >}}. The list of resources ...

nit: What is the most important column? The text in the first two columns are emphasized; the last three columns are not.

Reply:
Hmmm, I'm not familiar with the shortcodes, but I want to make it explicitly clear that this was only checked for this specific version.

The first two columns are most important, imo. The other three are more there as reference.

Reply:
@justaugustus Thanks for your patience! OK--I think we can get by without a shortcode here. (But note to self about creating one for specific versions...)

The [preview](https://deploy-preview-13223--kubernetes-io-master-staging.netlify.com/docs/reference/kubectl/overview/) looks good. Let's get this thing approved!